brainstormforce/astra
# AST-477 - fix: Alignwide section cuts the screen at exact breakpoint window

### Description
- Text gets cut off on at breakpoint window with align wide section

### Screenshots
- https://share.bsf.io/Blu48lDd

### Types of changes
- Bug fix

### How has this been tested?
- Import home page of this site - https://websitedemos.net/brandstore-08/
- Or add alignwide section on page 
- Check frontend at exact 921px breakpoint

### Checklist:
